documentation/openapi-authoring-automated-guidelines.md

```diff
@@ -525,26 +515,47 @@ Links: [Index](#index) | [Error vs. Warning](#error-vs-warning) | [Automated Rul
 
 **Why the rule is important**: Per the ARM guidelines, each swagger specification must contain one resource provider.
 
-**How to fix the violation**: N/A.
+**How to fix the violation**: One swagger specification must have one resource provider. Feel free to create multiple swaggers, each for one provider.
```

Comment:
Adding link to https://github.com/Azure/autorest/blob/185e337137c990b9cc1b8ebbb272e76eeeef43a1/docs/user/literate-file-formats/configuration.md as well